vmck/backends/qemu.py

```diff
@@ -19,8 +19,8 @@ def resources(vm_port, options):
     }
     return {
         'Networks': [network],
-        'MemoryMB': options['memory'],
-        'CPU': options['cpu_mhz'],
+        'MemoryMB': int(options['memory']),
```

Comment:
Again, these should already be integers, don't attempt to convert them, it only hides bugs.